packages/loader/container-loader/src/containerContext.ts

```diff
@@ -277,6 +277,10 @@ export class ContainerContext implements IContainerContext {
         this.runtime.setAttachState(AttachState.Attaching);
     }
 
+    setBlobRedirectTable(table: Map<string, string>) {
+        (this.runtime as any).setBlobRedirectTable(table);
```

Comment:
This casting should be fixed by changing definitions. Also it should be optional (and container fail if it's not there with some more reasonable error message than just "can't access setBlobRedirectTable on undefined").
We could make it non-optional, but it seems a bit against our principles on how we keep changes between loader & container runtime (even though it is for scenario that does not yet exists)